packages/laconia-batch/src/S3ItemReader.js

```diff
@@ -1,14 +1,13 @@
 const get = require("lodash.get");
+const { GetObjectCommand } = require("@aws-sdk/client-s3");
 const _ = { get };
 
-const extractS3JsonBody = data => {
-  const fileContent = data.Body.toString();
+const extractS3JsonBody = async data => {
   try {
-    return JSON.parse(fileContent);
+    const jsonString = await data.Body.transformToString();
```

Comment:
nitpick (non-blocking): I know you changed the error message to contain the actual original exception, but maybe this line should still be outside the try-catch?